search.ipynb

```diff
@@ -1346,23 +1367,63 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "d55324f7343a4c71a9a2d4da6d037037"
-      }
+       "model_id": "4c1f644bb8914a0bb67be8058eae9a50",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "<p>Failed to display Jupyter Widget of type <code>interactive</code>.</p>\n",
```

Comment:
You have some Jupyter Widget errors. I think you are missing a dependency. If you can't fix it, it would be best if you hid those errors.